packages/ketcher-react/src/script/ui/dialog/toolbox/labeledit.jsx

```diff
@@ -66,14 +67,28 @@ function deserialize(value) {
 
 function LabelEdit(props) {
   const init = { label: props.letter || serialize(props) }
-  const { formState, ...prop } = props
+  const { formState, onOk, ...prop } = props
```

Comment:
as we extract onOk prop here, it won't get into params prop in Dialog
it should affect functionality

Reply:
Deleted unused prop